### DISCUSSION TITLE
Release 2.9.0 is coming up

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enrouteText#38] [Team Posts]
Dear all,

the next release is coming up; the moving map can now display "gliding-sectors" in airspaces. As always: thank you for your help! Could you please fill in the last remaining translations? -- Thanks!

Best,

Stefan.

**Comment by Ivan081** [comment]
I return from hollyday on monday

Il sab 3 lug 2021, 14:18 Stefan Kebekus ***@***.***> ha
scritto:
